### PR TITLE
Populated Community tab using react

### PR DESCRIPTION
**Pull request by Calinator444**
Sorry I closed the previous request I made. I was reverting some commits I made and the request closed automatically. I've removed the packages and applied the feedback given.

### CODE REVIEW

File: src/js/communities/community-data.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+export const CommunityData = [{
+    name: 'Newcastle Coders Group',
+    location: "ICT Building, University of Newcastle",
+    period: {
+        interval: "Monthly",
+        //maybe use a single digit here and convert it later (i.e. "2" becomes "2nd")
```
Comment (clbaita) [inline src/js/communities/community-data.js:6]: We can probably remove this as to not cause confusion, I am guessing this is just train of thought.

File: src/js/communities/community-data.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        day: "2nd",
+        weekDay: "Wed"
+    },
+    //some of these groups only have a handful of links
```
Comment (clbaita) [inline src/js/communities/community-data.js:10]: Could you please move these comments to where the links are actually generated maybe line 17 in the community card file [here](https://github.com/newwwie/newwwie.com/pull/81/files#diff-9a2b79eb717319911e2ecb2ab242be2d0008c77525496d01b604bc3ae11b2958R17) just so it makes sense in context.

**Review by clbaita — CHANGES_REQUESTED** [review]
Well done on this and thank you for persisting. Just a couple of things with the comments and we should be good to go.

Hopefully this should make the communities tab easier to update. 👍

File: src/js/communities/community-data.js (hunk 2 above)
Comment (Calinator444) [reply src/js/communities/community-data.js:10]: Oh sorry, I thought I removed those.

File: src/js/communities/community-data.js (hunk 1 above)
Comment (Calinator444) [reply src/js/communities/community-data.js:6]: Oh sorry, I thought I removed that comment.

File: src/js/communities/community-data.js (hunk 2 above)
Comment (Calinator444) [reply src/js/communities/community-data.js:10]: Yeah that's fair. I'll fix the formatting as well.

**Review by clbaita — APPROVED** [review]
Looks good now, thanks again. :rocket:

**Pull request merged**